src/nvim/options.lua

```diff
@@ -30,6 +30,7 @@
 --- @field cb? string
 --- @field expand_cb? string
 --- @field tags? string[]
+--- @field available_values? table
```

Comment:
Is there a more conventional name that can be used? There are many such examples in the codebase.

Reply:
I would prefer `valid_values`